ktor-client/ktor-client-tests/common/test/io/ktor/client/tests/features/CacheTest.kt

```diff
@@ -367,4 +428,26 @@ class CacheTest : ClientLoader() {
             assertSame(publicCacheEntry, publicCache().first())
         }
     }
+
+    /**
+     * Does delay and ensures that the [GMTDate] measurements report at least
+     * the specified number of [milliseconds].
+     * The reason why it's not the same is that on some platforms time granularity of GMTDate
+     * is not that high and could be even larger than a millisecond.
+     */
+    private suspend fun delayGMTDate(milliseconds: Long) {
+        var delayValue = milliseconds
+
+        do {
+            val start = GMTDate()
+            delay(delayValue)
+            val end = GMTDate()
+            if (end > start && end > start + milliseconds) {
```

Comment:
do you need `end > start` here?

Reply:
Well, in the test we don't need this fast-path, you are right, will remove it 👍🏻